src/amo/components/HeroRecommendation/styles.scss

```diff
@@ -104,3 +104,35 @@
     text-decoration: none;
   }
 }
+
+@mixin gradientStyle($stopColor, $startColor: $color-ink-80) {
+  background-image: linear-gradient($startColor, $stopColor);
+
+  .HeroRecommendation-link {
+    &:hover,
+    &:active {
+      background-color: $stopColor;
+    }
+  }
+}
+
+// Styles that correspond to gradients returned by the hero API
+.color-ink-80-color-blue-70 {
+  @include gradientStyle($color-blue-70);
```

Comment:
Why not just pass the start color explicitly? Example:

```css
@include gradientStyle($color-ink-80, $color-blue-70);
```

I see that you changed the mixin to accept `$start-color` but I don't see why these calls should rely on the default; the class name is already explicitly calling for `color-ink-80`.

Reply:
True, but that just seems like unnecessary code. Isn't the reason to define a default value so you don't have to pass it every time you want to use it? We now have the ability to pass a start color if we need to, but we don't currently need to. If any of this changes on the API side we're going to have to define new selectors anyway, so I'm not sure why we should add code now to pass in what we know is the default value.

I don't feel that strongly about it, so if you do I'm happy to change it, but I really don't think it's necessary.

Reply:
Yes, thanks!